File: Astromaximum/Classes/utils/AmaxTimezoneTransition.m
//
//  AmaxTimezoneTransition.h
//  Astromaximum
//
//  Created by admin on 15.11.12.
//  Copyright (c) 2012 S&W Axis. All rights reserved.
//

#import "AmaxTimezoneTransition.h"

@implementation AmaxTimezoneTransition

- (id)initWithTime:(NSDate *)aTime offset:(NSTimeInterval)aOffset name:(NSString *)aName
{
    time = aTime;
    offset = aOffset;
    name = aName;
    return self;
}

@end

File: Astromaximum/Classes/utils/AmaxTimezoneTransition.swift
//
//  AmaxTimezoneTransition.swift
//  Astromaximum2
//
//  Created by Andrei Ivushkin on 12.07.2020.
//  Copyright © 2020 S&W Axis. All rights reserved.
//

import Foundation

@objcMembers class AmaxTimezoneTransition : NSObject {
    var time:Date
    var offset:TimeInterval
    var name:String
    
    init(time:Date, offset:TimeInterval, name:String) {
        self.time = time
        self.offset = offset
        self.name = name
    }
}
